python/hail/docs/types.rst

```diff
@@ -29,7 +29,7 @@ a :class:`.tstruct` with each option:
 .. autosummary::
     :nosignatures:
 
-    Type
+    HailType
```

Comment:
This change is because Type is a construct in the `typing` stdlib module, and it causes headaches to shadow that.

Reply:
I think this also caused the sphinx errors